cognite_toolkit/cdf_tk/load/_resource_loaders.py

```diff
@@ -391,6 +396,20 @@ def get_id(cls, item: Function | FunctionWrite) -> str:
             raise ValueError("Function must have external_id set.")
         return item.external_id
 
+    def load_resource(
+        self, filepath: Path, ToolGlobals: CDFToolConfig, skip_validation: bool
+    ) -> FunctionWrite | FunctionWriteList | None:
+        functions = load_yaml_inject_variables(
+            filepath, ToolGlobals.environment_variables(), required_return_type="list"
+        )
+        for func in functions:
+            self.extra_configs[func["externalId"]] = func.pop("extraConfigs", {})
```

Comment:
`extraConfigs` feels a bit generic. Is there another name that could make it clear to the user that this is extra information used by the toolkit only? 

`cdfTkConfig` or `toolkitConfig`?

Reply:
reverting